x-pack/solutions/observability/plugins/infra/public/components/asset_details/components/expandable_content.tsx

```diff
@@ -19,24 +19,35 @@ interface ExpandableContentProps {
 export const ExpandableContent = (props: ExpandableContentProps) => {
   const { values } = props;
   const [isExpanded, toggle] = useToggle(false);
+  const showLessRef = useRef<HTMLAnchorElement | null>(null);
+  const showMoreRef = useRef<HTMLAnchorElement | null>(null);
 
   const list = Array.isArray(values) ? values : [values];
   const [first, ...others] = list;
   const hasOthers = others.length > 0;
   const shouldShowMore = hasOthers && !isExpanded;
 
+  const handleToggle = () => {
+    toggle();
+
+    setTimeout(() => {
```

Comment:
Q: why is the timeout of 0 needed here?

Reply:
Now I remember why I used the timeout, I changed to use useEffect() but need to add another state check to avoid the focus to be always set on show more / less on reload.